impeller/docs/faq.md

```diff
@@ -100,3 +100,113 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
+* Why did the Flutter team build Impeller?
+  * The short answer, for consistent performance. The longer answer...
```

Comment:
minor... do we need to preface each section below with a `*` ? I read the below as a series of paragraphs, not bullets.